docs/reference/query-dsl/mlt-query.asciidoc

```diff
@@ -80,7 +81,8 @@ GET /_search
                         "first": "Ben",
                         "last": "Grimm"
                     },
-                    "_doc": "You got no idea what I'd... what I'd give to be invisible."
+                    "_doc": "You got no idea what I'd... what I'd give to be
+invisible."
```

Comment:
I think the extra line break introduced here is what's causing the doc test error.